batch.go

```diff
@@ -109,34 +109,24 @@ func (c *Client) AddShipmentsToBatch(batchID string, in ...*Shipment) (out *Batc
 // AddShipmentsToBatch, but allows specifying a context that can interrupt the
 // request.
 func (c *Client) AddShipmentsToBatchWithContext(ctx context.Context, batchID string, in ...*Shipment) (out *Batch, err error) {
-	req := batchRequest{Batch: &Batch{Shipments: in}}
+	req := map[string]interface{}{"shipments": in}
 	err = c.post(ctx, "batches/"+batchID+"/add_shipments", req, &out)
 	return
 }
 
 // RemoveShipmentsFromBatch removes shipments fro, an existing batch, and
 // returns the updated batch object.
-func (c *Client) RemoveShipmentsFromBatch(batchID string, shipmentIDs ...string) (out *Batch, err error) {
-	req := batchRequest{
-		Batch: &Batch{Shipments: make([]*Shipment, len(shipmentIDs))},
-	}
-	for i := range shipmentIDs {
-		req.Batch.Shipments[i] = &Shipment{ID: shipmentIDs[i]}
-	}
+func (c *Client) RemoveShipmentsFromBatch(batchID string, in ...*Shipment) (out *Batch, err error) {
```

Comment:
Is it possible to pass in shipment IDs like our documentation shows?

Reply:
After some deliberation, we are going to use `interface{}` so the user can pass in either `Shipment IDs` or `Shipment structs`, this will avoid breaking changes.